plugins/CoreVisualizations/JqplotDataGenerator/Chart.php

```diff
@@ -27,6 +27,8 @@ class Chart
     // temporary
     public $properties;
 
+    public $dataTable;
```

Comment:
I think this property should not be needed. Instead I think we should remove any usage of that property in other files. If I remember correctly it was only set in some other files but actually used anywhere else... Didn't I remove the usages in a previous PR?

Reply:
Yes, this was fixed by your previous PR, reverted. :+1: